account_due_list/migrations/8.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# © 2016 Savoir-faire Linux
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+column_drops = {
+    ('account_move_line', 'day'),
+}
+
+
+@openupgrade.migrate()
+def migrate(cr, version):
+    if not version:
```

Comment:
@pedrobaeza @jbeficent shouldn't this file be removed from version 9 of the module?